ravendb/resource_ravendb_server.go

```diff
@@ -476,6 +911,67 @@ func resourceServerRead(ctx context.Context, d *schema.ResourceData, meta interf
 	return nil
 }
 
+func flattenDatabases(databases []Database) []map[string]interface{} {
+	tfs := make([]map[string]interface{}, 0)
+	for _, db := range databases {
+		tf := map[string]interface{}{
+			"database": []map[string]interface{}{
+				{
+					"name":              db.Name,
+					"settings":          db.Settings,
+					"replication_nodes": db.ReplicationNodes,
+					"encryption_key":    generateHash256(db.Key),
+					"indexes":           flattenIndexes(db.Indexes),
+				},
+			},
+		}
+		tfs = append(tfs, tf)
+	}
+	return tfs
+}
+
+func generateHash256(encryptionKey string) string {
```

Comment:
Let's use `Argon2id` instead, see: https://pkg.go.dev/golang.org/x/crypto/argon2

This is not technically needed, but better to have extra level of security here.
Also, probably need to call this `encryption_key_hash`

Reply:
done